src/renderer/src/components/common/Series/SeriesSidebar.tsx

```
@@ -0,0 +1,60 @@
+import { useSeriesStore } from '@renderer/stores/seriesStore'
+
+const SeriesSidebar = () => {
+  const currentSeriesPath = useSeriesStore((state) => state.currentSeriesPath)
+  const [treeData, setTreeData] = useState<TreeNode[]>([])
+
+  console.log(currentSeriesPath)
+
+  useEffect(() => {
+    if (!currentSeriesPath) return
+
+    const fetchTree = async () => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)